content/en/events/2023/kcscn/schedule.md

```diff
@@ -10,7 +10,7 @@ weight: 3
 
 A [CFP] for sessions is open through Sunday, September 10th.
 
-[CFP]: https://wj.qq.com/s2/12996651/2260/
+[CFP] is closed.
```

Comment:
Please just remove the whole CFP section instead of doing this.  It's got a broken link, and is confusing to attendees.

Reply:
Thanks for reviewing.

CFP part is removed now.

And add some links for speakers.